src/styles/styles.css

```diff
@@ -0,0 +1,315 @@
+@charset "UTF-8";
+
+/* VARIABLES HEADER */
+
+/* VARIABLES CARD */
+
+body {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: Roboto, sans-serif;
```

Comment:
The `font-family` property should be defined in a variable. This helps to maintain the code and makes it easier to change the font in the future if needed.